week-4/F-object-keys/exercise-part-0.js

```diff
@@ -0,0 +1,32 @@
+/*
+
+Return the keys of the following object
+
+*/
+
+var capitalCities = {
+  edinburgh: 'Scotland',
+  nairobi: 'Kenya',
+  canberra: 'Australia',
+  ottowa: 'Canada'
+};
```

Comment:
I think this should be reversed. When I see an object "capitalCities", I'd expect to pass it a country, and get its capital city: `console.log(capitalCities['Scotland']);`

It also seems a bit more useful that way, as you usually know the country, but need to find out the capital city, rather than the other way around.